### DISCUSSION TITLE
2023/07/local-perspective/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/xiangyun#87] [General]
# 2023/07/local-perspective/

https://xiangyun.rbind.io/2023/07/local-perspective/

**Comment by earfanfan** [comment]
艺术是主观的，但并不是虚幻缥缈的，梵高的画作最为人称道的是极有生命力，我读过梵高书信集，字里行间能看出来他整个人本来也是充满生命力的。从一名热爱艺术的创作者的角度来说，不论是创作一幅画还是创作一首曲子抑或其他形式，创作的动机首先会是记录、表达、传递一种情绪或情感，而后能看到或听到作品的人才能体会到创作者所表达的情绪或情感，进而产生共鸣、理解。人心里产生的那些情绪或情感是极为珍贵的，大多数人都会经历但无法捕捉，只有那些创作者能够运用各种艺术创作的形式捕捉到。AI 只是为了完成人类的指令，尽管可以模仿人类的情绪或情感，但无法替代。

你所说的“讨好型人格”，会不会在离家这么多年的成长中已经重塑了但不自知呢，反正你跟我交流的时候一向是该说说，我可从来没感觉到撒时候你有“讨好”我或者让着我。

**Reply by XiangyunHuang** [reply]
咱们理解的应该是差不多的，我的用词「虚幻缥缈」可能不太恰当，我想表达的是只能去感受美，我感受到的美很难描述给你，让你也感受到我感受到的那种美。如果美是可以简单传递和复制的，那么从达芬奇、梵高等等一路下来，人类艺术水平应该是大踏步地前进了，可是现在的人仿制都困难（同时代也没有出来很多像达芬奇那样艺术水平的画家），更别说超越。

接下来，我有一连串的问题。深度学习、AI 大模型可以容纳人类知识，自然科学就不用说了，艺术性的东西是不是也可以容纳，复制、传递是 AI 的强项，所以，归结下来，美是不是可以被计算机编码？它能否自成画派，达到或超越现有的艺术水平？如果它能达到人类艺术水平，只要给算力，迭代是 AI 与生俱来的能力，它肯定会超越？迭代越多，超越得越多？如果艺术可以如此超越，其它领域是不是一下子也能迅速超越，然后，AI 远远地超越人类。多少年了，我觉得人类社会没有突变过，一直是缓慢地前进或倒退。最近，一次次的 AI 进展，给人感觉发生了突变，大踏步地往前进化，这种进化有限制吗？它会始终受制于人吗？

因为我们是平辈论交，没有上下级、尊卑、利益关系呀。

**Reply by earfanfan** [reply]
哈哈，你不先定义一下何为美吗？而且其实美和审美观还是两个东西呢。

既然你认为你能感受到的美是你的主观感受，那么所有人的主观感受能够一致吗？艺术性的东西里面确实有一部分是可以传递出来的，就是那些技巧之类的啊。如果AI进化的终点是拥有和懂得了人类的情感，那么真得有可能脱离于人的掌控和利用吧。

**Reply by XiangyunHuang** [reply]
> 哈哈，你不先定义一下何为美吗？而且其实美和审美观还是两个东西呢。

你这里考虑的是美的多样性的问题，这一点我不否认。同样的，美也有它的独特性，当然还有共同性，比如对称性，曲线美等等。既有规范的，也有不遵循规范的。现在的问题是 AI 能不能编码任意一种美？比如下面两种美，我希望 AI 不是学技巧、画风，而是给人的同一范畴的美感，它不是仿制下面的画作，而是另一个继梵高、莫奈之后的 AI 印象派大师，对印象派有继承有发展，而且得是人能欣赏得了的。突然，我想到个问题，目前，AI 和人交流都能被发现是机器人，这再通过第三方的画作与人交流，是不是要求 AI 进化到更高层次的智能呢？

![戴珍珠耳环的少女](https://github.com/rbind/xiangyun/assets/12031874/15f05487-def6-4fa3-a744-e83a158f2250)

![日出·印象](https://github.com/rbind/xiangyun/assets/12031874/058855e4-9745-4da1-a90f-cc210f3b7695)

**Reply by earfanfan** [reply]
你在哪里搞的这两张高清图啊，上面那张少女的脸简直像是那种冰裂纹的瓷器。我记得看油画理论上是要隔点距离去欣赏的说，这太高清了，一怼到我的狗眼上，不属于画作本身的瑕疵一块被放大了。

>现在的问题是 AI 能不能编码任意一种美？

这是个问题嘛……你确定艺术和美这种问题也得坚持追求一下全面和客观么，我们做数据分析时追求的全面和客观心态是十分务实的，不适合迁移应用到任何主题下面啊，俺的个人想法。

>AI 和人交流都能被发现是机器人，这再通过第三方的画作与人交流，是不是要求 AI 进化到更高层次的智能呢？

谁提的要求？为撒要提这个要求？之前我看新闻说是有的AI通过了图灵测试来着，并没有被发现是人，除非它自爆其短。

思考完这些问题，记得回到具体的生活中去啊喂。本阳间作息者滚去睡了zzz

**Reply by XiangyunHuang** [reply]
以上两张图一点也不高清，很糟糕，还没我的画册好看，只是从维基百科词条里抠出来用一下。

> 现在的问题是 AI 能不能编码任意一种美？

难道又是一句有歧义的话？我想表达的是在万千种美中，AI 能不能表达出来其中一种美。算了，我也困了，下线。

**Comment by XiangyunHuang** [comment]
我最初也是这样认为的，后来，也就是昨天，我又去中国美术馆看画展了，其实我觉得还有很多看法，至少有这么三种。远看，画连同布展特意为它准备的光影环境，这很容易勾起观看者的注意力。近看，去掉光影环境，整幅画映入眼帘，观看画的各个部分，看结构、组织和平衡性，以及带给人的一些遐想。超近看，近看发现的画作中有意思的局部，以及画家绘画留下的笔法痕迹，所用纸张生熟特点。这可以看出画家用笔的力道，有的刚劲，有的灵动，还有水和墨交融的细节。

**Reply by earfanfan** [reply]
这是当然，当我们所讨论的内容不涉及一个具体场景时，我们本就不用秉承什么具体问题具体分析的思路去思考问题，而是可以随意思随意想。虽然你所说的结构、组织、力道、笔法这些我全都不懂，但看画本来就不用局限于隔的远近，我记得以前看过一部电视剧，有一个情节大致是说对于一幅画来说，用更合适的画框也更好，大概是画框也能增强画的某种效果吧，所以当绘画成为一种艺术后，连带着装裱画、制作画框也可成为一门艺术。

你原先提到的问题，我现在看明白了。你是问AI绘画是不是可以不仅仅做到复制（风格、技巧），是不是还可以通过大量的学习继承艺术并且跨过人类的有限造成的瓶颈，开创出更加能被人类接受和认可的风格、审美观念。如果这次我没又理解错的话，我还是觉得除非AI能理解人类的情绪、情感、思想。每一个艺术大家，不论是哪种艺术形式，在他们熟练掌握了技术后，此后他们本身的不断进步就是和思想变迁息息相关。尽管我一个流派都不认识，但我觉得也许每个流派背后用于创造艺术作品的核心思想都可以转换成一种人生观、世界观、生活态度。

又尽管，现在艺术形式已经十分丰富，各类艺术技法的发展已经使得普通人渐渐无法欣赏那些作品本身，但无论如何，创造艺术本身在于表达，如果AI不想要表达什么或者说它的表达不完整、只是从别处借来缝补在一起的，那它创作的艺术价值又在哪里呢。

以上是一个纯纯门外汉的浅薄愚见。

**Reply by XiangyunHuang** [reply]
> 你原先提到的问题，我现在看明白了。你是问AI绘画是不是可以不仅仅做到复制（风格、技巧），是不是还可以通过大量的学习继承艺术并且跨过人类的有限造成的瓶颈，开创出更加能被人类接受和认可的风格、审美观念。如果这次我没又理解错的话，我还是觉得除非AI能理解人类的情绪、情感、思想。每一个艺术大家，不论是哪种艺术形式，在他们熟练掌握了技术后，此后他们本身的不断进步就是和思想变迁息息相关。尽管我一个流派都不认识，但我觉得也许每个流派背后用于创造艺术作品的核心思想都可以转换成一种人生观、世界观、生活态度。

我觉得你表达得比我好。

**Reply by XiangyunHuang** [reply]
我这周末在整理观后感，刚放出来 3 篇，欢迎有空来交流，今晚还去看昆曲《西厢记》，我期待有新的收获。

**Reply by earfanfan** [reply]
哇嗷，我一会去看。我也期待你有新的感想，回头再跟我这撒也不懂的白丁科普一下。

**Reply by XiangyunHuang** [reply]
感想很简短。感觉没有上次好，可能是自己坐在在3层左侧很偏的位置，而且我的左后方一直有人断断续续地交头接耳，我的右前方有人时不时拿手机拍照、录像。《西厢记》比较热闹，名气比《烂柯山》大，人很多，年轻人的比例很高。时间是 2 个半小时，这个座位坐久了感觉不舒服，没有话剧院的舒服。

**Reply by earfanfan** [reply]
为撒年轻人比例高呢，我按“昆曲 年轻人”搜了一下，原来昆曲年轻化变革有好几年了，只是我不知道。另，有人拿手机拍照录像居然没有工作人员来干涉制止一下么，约莫好久以前我去音乐厅听那个交响乐，当时想举起手机拍一下大厅那个超级大的圆盘子吊顶，还没调好角度就被工作人员拿那种射出红点的灯提醒制止了。至于有人出声交流这种嘛，哈，我一向自认嘴炮能力差、吵架功力不足，就连坐高铁上旁边座位小孩一直踢腿都会选择忍。

**Reply by XiangyunHuang** [reply]
> 为撒年轻人比例高呢

我觉得跟曲目有关系，这个《西厢记》讲年轻人的爱情故事。

> 还没调好角度就被工作人员拿那种射出红点的灯提醒制止了。

射出红点的灯，这里也有哈，我看到了，但我没想到是制止，还以为又是哪个不懂规矩的在瞎晃呢。
